app/src/main/java/org/thoughtcrime/securesms/webrtc/NetworkChangeReceiver.kt

```diff
@@ -10,9 +11,18 @@ import org.session.libsignal.utilities.Log
 
 class NetworkChangeReceiver(private val onNetworkChangedCallback: (Boolean)->Unit) {
 
+    companion object {
+
+        // Method to check if a valid Internet connection is available or not
```

Comment:
```suggestion
        /**
        * @return true if a valid internet connection is available.
        */
```

is a bit clearer, it says what it returns in which case, we're not checking/asserting the value, so `check` is misleading, if it's `available or not` makes it less clear when it returns what, e.g. to take it to a logical extreme: `returns true if x is available or not` would just `return true`.

Or we could just delete the comment, as the method name is informative  if we rename it `hasValidNetworkConnection`? maybe even a property (:

And it may as well be a top-level extension function on `Context`.

then usage becomes...

```
context.hasValidNetwokrConnection
```

instead of 

```
NetworkChangeReceiver.hasValidNetwokrConnection(context)
```

which doesn't really have anything to do with `NetworkChangeReceiver` anyway. as `NetworkChangeReceiver` does not impact the value we retrieve in any way.

Reply:
Rephrased to `determine` rather than `check` - also, the method has moved to a new `NetworkUtils` class at Thomas' prompting.